apps/client/src/components/StepByStepNavigation/StepByStepNavigation.tsx

```diff
@@ -8,21 +8,25 @@ import StepByStepNavigationStyle, { Props } from "./StepByStepNavigationStyle";
 const StepByStepNavigation: React.FC<
   Props & React.HTMLAttributes<HTMLElement>
 > = ({
-  children: childrenProps,
+  children: childrenProp,
   customSize,
   disabledTextColor,
   doneTextColor,
   highlightActive,
   ...otherProps
 }) => {
-  const { children } = usePassPropsToChildren(childrenProps, {
+  // Pass the props defined below to all the children
+  const { children } = usePassPropsToChildren(childrenProp, {
     customSize,
     disabledTextColor,
     doneTextColor,
     highlightActive,
   });
 
-  if (!childrenProps) return null;
+  // React Hook "usePassPropsToChildren" must be called before this `return` statement
```

Comment:
Or we can use useEffect() 🙄

Reply:
What if you change the import to `import passPropsToChildren from "../../utils/usePassPropsToChildren";`?

Reply:
Good point! Works

Reply:
Fixed https://github.com/Amsterdam/vergunningcheck/pull/440/commits/bdd53468db1949504cda9c5fdc975bec0298e21e